.github/workflows/lint.yml

```diff
@@ -28,7 +28,26 @@ jobs:
         ./setup.sh
         ./setup-dev.sh
 
+
     - name: Run linters
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+
+    - name: Activate virtualenv
+      run: |
+        source ~/.venv/bin/activate
```

Comment:
Where should I source this from? this isn't working.

Reply:
Afair it should be `source ./.venv/bin/activate` since venv is created in the pwndbg directory if `PWNDBG_VENV_PATH` is not set.

Reply:
I just hope that the PATH and other env changes made by this will be propagated to the mypy step.